locations/spiders/paul_za.py

```
@@ -1,12 +1,22 @@
+from locations.items import Feature
 from locations.spiders.paul_fr import PAUL_SHARED_ATTRIBUTES
-from locations.storefinders.yext_search import YextSearchSpider
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class PaulZASpider(YextSearchSpider):
+class PaulZASpider(SitemapSpider, StructuredDataSpider):
     name = "paul_za"
     item_attributes = PAUL_SHARED_ATTRIBUTES
-    host = "https://location.paulsa.co.za"
+    sitemap_urls = ["https://locations.paulsa.co.za/sitemap.xml"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)